guides/micronaut-jms-aws-sqs/java/src/test/java/example/micronaut/TestSqsClientFactory.java

```diff
@@ -0,0 +1,30 @@
+package example.micronaut;
+
+import com.amazonaws.client.builder.AwsClientBuilder;
+import com.amazonaws.regions.Regions;
+import com.amazonaws.services.sqs.AmazonSQS;
+import com.amazonaws.services.sqs.AmazonSQSClientBuilder;
+import io.micronaut.aws.sdk.v1.EnvironmentAWSCredentialsProvider;
+import io.micronaut.context.annotation.Factory;
+import io.micronaut.context.annotation.Requires;
+import io.micronaut.context.env.Environment;
+import jakarta.inject.Singleton;
+
+import java.util.Optional;
+
+@Factory
+@Requires(env = Environment.TEST)
```

Comment:
This is redundant since the class belong to the test sources, so no way it can be loaded outside tests.

Instead, add `@Replaces(SqsClientFactory.class)`